src/main/java/pl/agh/edu/master_diet/controller/CalculatorController.java

```diff
@@ -0,0 +1,32 @@
+package pl.agh.edu.master_diet.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.agh.edu.master_diet.core.model.rest.calculator.bmi.CalculateBmiRequest;
+import pl.agh.edu.master_diet.core.model.rest.calculator.bmi.CalculateBmiResponse;
+import pl.agh.edu.master_diet.core.model.shared.BmiParameters;
+import pl.agh.edu.master_diet.service.CalculatorService;
+import pl.agh.edu.master_diet.service.converter.ConversionService;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/calculator")
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class CalculatorController {
+
+    private final CalculatorService calculatorService;
+    private final ConversionService conversionService;
+
+    @PostMapping("/bmi")
```

Comment:
it's shortcut from Body Mass index should be BMI

Reply:
Fixed